test/expr/unary/selector/selector.swift

```diff
@@ -116,7 +116,7 @@ func testParseErrors3(_ c1: C1) {
 
 func testParseErrors4() {
   // Subscripts
-  _ = #selector(C1.subscript) // expected-error{{type 'C1.Type' has no subscript members}}
+  _ = #selector(C1.subscript) // expected-error{{type 'C1' has no member 'subscript'}}
```

Comment:
I'd say that `C1.subscript` will always be invalid unless you declare a ``func `subscript`() {}``. Otherwise we would get into ambiguity issues if you declare both a subscript and a function named `subscript`.